contracts/ERC677MultiBridgeToken.sol

```diff
@@ -0,0 +1,108 @@
+pragma solidity 0.4.24;
+
+import "./ERC677BridgeToken.sol";
+
+/**
+ * @title ERC677MultiBridgeToken
+ * @dev This contract extends ERC677BridgeToken to support several bridge simulteniously
+ */
+contract ERC677MultiBridgeToken is ERC677BridgeToken {
+    address public constant F_ADDR = 0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF;
+    uint256 internal constant MAX_BRIDGES = 50;
+    mapping(address => address) public bridgePointers;
+    uint256 public bridgeCount;
+
+    event BridgeAdded(address indexed bridge);
+    event BridgeRemoved(address indexed bridge);
+
+    constructor(string _name, string _symbol, uint8 _decimals) public ERC677BridgeToken(_name, _symbol, _decimals) {
+        bridgePointers[F_ADDR] = F_ADDR; // empty bridge contracts list
+    }
+
+    /**
+    * @dev Adds one more bridge contract into the list
+    * @param _bridge bridge contract address
+    */
+    function setBridge(address _bridge) external onlyOwner {
+        require(bridgeCount < MAX_BRIDGES);
+        require(AddressUtils.isContract(_bridge));
+        require(!isBridge(_bridge));
+
+        address firstBridge = bridgePointers[F_ADDR];
+        require(firstBridge != address(0));
+        bridgePointers[F_ADDR] = _bridge;
+        bridgePointers[_bridge] = firstBridge;
+        bridgeCount = bridgeCount.add(1);
+
+        emit BridgeAdded(_bridge);
+    }
+
+    /**
+    * @dev Removes one existing bridge contract from the list
+    * @param _bridge bridge contract address
+    */
+    function unsetBridge(address _bridge) external onlyOwner {
```

Comment:
As it was proposed above for renaming `setBridge` to `addBridge`, maybe rename `unsetBridge` to `removeBridge`?